### PR TITLE
Feature/wdp190703 29/section brands markup and style

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**
I added section BRANDS Markup and styles, is not RWD yet because that is different task

### CODE REVIEW

File: src/partials/60_section-gallery.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h5>Aenean Ru Bristique</h5>
               <img src="images/WoodChair.png" alt="WoodChair" />
             </div>
             <div class="gallery-slide-box">
-              <img src="images/OfficeChair.png" alt="OfficeChai" />
+              <img src="images/OfficeChair.png" alt="OfficeChair" />
```
Comment (matwjablonski) [inline src/partials/60_section-gallery.html]: alt should be a description ;)

File: src/sass/components/_section--brands.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        a {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          color: $white;
+          width: 100%;
+          height: 100%;
+          text-decoration: none;
+
+          &:hover {
```
Comment (matwjablonski) [inline src/sass/components/_section--brands.scss]: hover only on supporting devices

File: src/partials/60_section-gallery.html (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/partials/60_section-gallery.html]: Now is description :)

File: src/sass/components/_section--brands.scss (hunk 2 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--brands.scss]: added !

**Pull request merged**